### ISSUE TITLE
status-go to show "node offline" when device is offline

### ISSUE THREAD

**Issue by sla-shi** [status-im/status-go#242]
TBD by @tiabc

**Comment by tiabc** [comment]
It seems `status-go` responds with `no running geth node detected` instead of `node offline` when the node is offline. If so, it needs fixing: non-running node is different from an offline node.

@rasom, could you, please, add your requirements if there are any and write which `status-go` signals `status-react` calls from the moment of application start until the error in question?

**Comment by sla-shi** [comment]
This issue is linked to https://github.com/status-im/status-react/issues/1295

**Comment by MarinX** [comment]
#250

**Comment by tiabc** [comment]
Taking over.

**Comment by adambabik** [comment]
I checked how the node behaves when there is no network and it works just fine. It does not report any issues, it just sits and waits. When the connection is back it starts synchronisation. 

So, I started thinking that it shouldn't really be status-go's job to figure out if there is the Internet connection at the moment but rather the app itself. Mobile OS-es have much reacher APIs to check that. 

Also, I am not seeing a reason why one could not start a node without having the Internet. You can start Messanger without the connection, it will just notify you about that with a message.

What do you think @sla-shi @tiabc?

**Comment by adambabik** [comment]
I can't reproduce this bug. I used status-go (https://github.com/status-im/status-go/commit/8fb2424ea564beddf42b522066c52ca670ec82a9) and status-react `feature/upgrade-status-go-g0270590`.

It works as expected, i.e. I can successfully log in even if the Internet connection is disabled. After logging in, I can see a banner "Offline". Tested with iOS Simulator with disabled network before running the app. After enabling the network, the banner disappears.
